### ISSUE TITLE
Seeking help with environment configuration

### ISSUE THREAD

**Issue by Denwey1** [tsinghua-fib-lab/LCSim#4]
Hi, Thanks for open-sourcing the awesome project! I encounter an issue when I run pip install waymo-open-dataset==1.0.1, and the details are as follows:
ERROR: Could not find a version that satisfies the requirement waymo-open-dataset==1.0.1 (from versions: none)
ERROR: No matching distribution found for waymo-open-dataset==1.0.1
The python version of my conda environment is 3.10.14 and I have successfully installed the rest of the contents of the requirments.txt as follows:
Package                      Version
---------------------------- -----------
absl-py                      2.1.0
aiohttp                      3.9.5
aiosignal                    1.3.1
astunparse                   1.6.3
async-timeout                4.0.3
attrs                        23.2.0
certifi                      2024.7.4
charset-normalizer           3.3.2
click                        8.1.7
cloudpickle                  3.0.0
contourpy                    1.2.1
cycler                       0.12.1
Farama-Notifications         0.0.4
filelock                     3.15.4
flatbuffers                  24.3.25
fonttools                    4.53.1
frozenlist                   1.4.1
fsspec                       2024.6.1
gast                         0.6.0
google-pasta                 0.2.0
grpcio                       1.64.1
gymnasium                    0.29.1
h5py                         3.10.0
idna                         3.7
Jinja2                       3.1.4
joblib                       1.4.2
jsonschema                   4.23.0
jsonschema-specifications    2023.12.1
keras                        3.4.1
kiwisolver                   1.4.5
libclang                     18.1.1
lightning-utilities          0.11.4
Markdown                     3.6
markdown-it-py               3.0.0
MarkupSafe                   2.1.5
matplotlib                   3.8.4
mdurl                        0.1.2
ml-dtypes                    0.3.2
mpmath                       1.3.0
msgpack                      1.0.8
multidict                    6.0.5
namex                        0.0.8
networkx                     3.3
numpy                        2.0.0
nvidia-cublas-cu12           12.1.3.1
nvidia-cuda-cupti-cu12       12.1.105
nvidia-cuda-nvrtc-cu12       12.1.105
nvidia-cuda-runtime-cu12     12.1.105
nvidia-cudnn-cu12            8.9.2.26
nvidia-cufft-cu12            11.0.2.54
nvidia-curand-cu12           10.3.2.106
nvidia-cusolver-cu12         11.4.5.107
nvidia-cusparse-cu12         12.1.0.106
nvidia-nccl-cu12             2.18.1
nvidia-nvjitlink-cu12        12.5.82
nvidia-nvtx-cu12             12.1.105
opt-einsum                   3.3.0
optree                       0.12.1
packaging                    24.1
pillow                       10.4.0
pip                          24.1.2
protobuf                     5.27.1
psutil                       6.0.0
Pygments                     2.18.0
pyparsing                    3.1.2
python-dateutil              2.9.0.post0
pytorch-lightning            2.2.1
PyYAML                       6.0.1
ray                          2.9.3
referencing                  0.35.1
requests                     2.32.3
rich                         13.7.1
rpds-py                      0.19.0
scikit-learn                 1.3.2
scipy                        1.14.0
setproctitle                 1.3.3
setuptools                   69.5.1
six                          1.16.0
sortedcontainers             2.4.0
sympy                        1.13.0
tensorboard                  2.16.2
tensorboard-data-server      0.7.2
tensorflow                   2.16.1
tensorflow-io-gcs-filesystem 0.37.1
termcolor                    2.4.0
threadpoolctl                3.5.0
torch                        2.1.2
torch_cluster                1.6.3
torch_geometric              2.5.3
torchmetrics                 1.4.0.post0
tqdm                         4.66.2
triton                       2.1.0
typing_extensions            4.12.2
urllib3                      2.2.2
Werkzeug                     3.0.3
wheel                        0.43.0
wrapt                        1.16.0
yarl                         1.9.4

**Comment by efyphil** [comment]
Try this waymo-open-dataset-tf-2-12-0

**Comment by Denwey1** [comment]
Thank you so much for your help！@efyphil
